### ISSUE TITLE
Preparar boilerplate para linguagem de back-end

### ISSUE THREAD

**Issue by andregumieri** [guminidigital/front-boilerplate#18]
Na minha idéia, seria o caso de ter uma pasta "www" para adicionar o código de backend. Também acho interessante estudarmos a integração com o ~~vagrant~~ docker.

**Comment by rafaelcavalcante** [comment]
@andregumieri, jogaríamos os CSS/JS processados pra dentro da WWW também?

**Comment by andregumieri** [comment]
Sim. Meio que www seria nosso dist. Pensando numa estrutura de wordpress, por exemplo, a pasta de distribuição seria www/wp-content/themes/nome-do-tema/
